bindings/c/tests/bdd.cpp

```diff
@@ -82,6 +82,14 @@ TEST_F(OpendalBddTest, FeatureTest)
     for (int i = 0; i < r.data->len; i++) {
         EXPECT_EQ(this->content[i], (char)(r.data->data[i]));
     }
+
+    // The blocking file should be deleted
```

Comment:
The BDD test should strictly follow the feature defined in `bindings/test`, which does not require a delete operation. To test this, we may need to have separate unit tests, but not in the `BDD.cpp`.

WDYT @Xuanwo

Reply:
This can be done in a new PR.

Reply:
Are you suggesting creating a new file specifically for testing the "delete" function? 🤔

Reply:
> Are you suggesting creating a new file specifically for testing the "delete" function?

Changes in this PR LGTM. We are discussing the BDD test for all bindings which is a new issue to address.

Reply:
Anyway, this review is out of the scope of this PR. I will mark this as resovled.

No change required.